assignments/arrays.js

```diff
@@ -74,16 +74,44 @@ console.log(`${inventory[49]['car_make']} ${inventory[49]['car_model']}`);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
+let carModels = [];
+let modelSort = inventory.sort(function(a,b) {
```

Comment:
This is great. You figured out how to pass custom callback to the `sort` method. It is imperative to note, however, that `sort` method alters the original array. As a result in this case the original data is no longer sorted by `id` as it used to be. 

A better strategy to have approached this challenge is to collect all car models in an array first and then sort this new array which contains only the strings of car models. The default behavior of sort is to sort alphabetically (make sure the array contains only text). So if we had collected all car models in the `carModels` array first, then in order to sort it all we would have had to do is to just call `carModels.sort()` without a callback. Putting the above-mentioned things into consideration you might end up with the following code.

```js
let carModels .= [];

for(let i = 0; i < inventory.length - 1; i++) {
  carModels.push(inventory[i].car_model);
}

console.log(carModels.sort());
```